float32[12] Q
float32[8] R
---
int16 status